components/automate/function/RunRowObjectResult.vue

```diff
@@ -45,31 +51,44 @@ const projectAccount = computed(() =>
 )
 const clientId = projectAccount.value.accountInfo.id
 
-const applicationIds = ref<string[]>([])
+const applicationIds = computed(() => {
+  // Old schema
+  if ('objectIds' in props.result) return getApplicationIdsFromOldSchema()
+  // New schema
+  return Object.values(props.result.objectAppIds).filter((id) => id !== null)
+})
 
 type Data = {
   applicationId?: string
 }
 
-// Loop over each objectId to run the query and collect application IDs
-props.result.objectIds.forEach((objectId) => {
-  const { result: objectResult } = useQuery(
-    objectQuery,
-    () => ({
-      projectId: props.modelCard.projectId,
-      objectId
-    }),
-    () => ({ clientId })
-  )
+const getApplicationIdsFromOldSchema = () => {
```

Comment:
I would remove support for old schema as it's not sustainable - like full excise it.